lib/compiler.js

```diff
@@ -268,6 +285,9 @@ Compiler.prototype.visitAttributes = function(attrs) {
         }
       } else if (attr.name === `id` && attr.val[0] === `'`) {
         id = attr.val;
+      } else if (attr.name === `attrs`) {
```

Comment:
not sure if this is the right level of specificity here but it works for our usage of snabbdom (which is probably the only one that really matters?)

Reply:
just to be safe, add it into the Snabbdom config and double-check here:
```suggestion
      } else if (this.serializeAttrsObjects && attr.name === `attrs`) {
```
(you'll need to add `serializeAttrsObjects` in both https://github.com/tdumitrescu/virtual-jade/blob/master/lib/config.js and https://github.com/tdumitrescu/virtual-jade/blob/02a6fd487da1287eb1de9318c25ccba914aecb0a/lib/compiler.js#L5-L9 to have it accessible as `this.serializeAttrsObjects` in this context)